.github/workflows/ci.yml

```
@@ -4,44 +4,54 @@ on:
       - name: Benchmark
         run: node benchmark
 
       - name: Codecov
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+
+      - name: Run codacy-coverage-reporter
+        # bad Windows -- https://github.com/codacy/codacy-coverage-reporter-action/issues/91
+        if: ${{ matrix.os !== 'windows-latest' }}
```

actionlint (1.7.4)

got unexpected character ' ' while lexing == operator, expecting '='

(expression)